apps/aecore/src/aec_chain.erl

```diff
@@ -143,7 +143,7 @@ init(_Args = [GenesisBlock]) ->
     %% Compute initial state of process.
     {ok, TopHeaderHash} = top_header_db_get(NewTopHeaderDb, ?TOP_HEADER),
     {ok, SerializedTopHeader} = headers_db_get(NewHeadersDb, TopHeaderHash),
-    {ok, TopHeader} = aec_headers:deserialize_from_network(SerializedTopHeader),
+    {ok, TopHeader} = aec_headers:deserialize_from_binary(SerializedTopHeader),
```

Comment:
Why rename? The network format is not necessarily binary, e.g. can be JSON. @sennui

Reply:
Network format is handled differently and inside network code. Request makers use deserializa_from_map, serialize_to_map. Swagger requires a map. My code has one function that invokes jsx.